themes/custom/droopler_theme/scss/components/_d_p_side_image.scss

```diff
@@ -22,23 +22,84 @@ $d-p-side-image-banner-text-padding: 7% !default;
 $d-p-side-image-banner-text-width: 35.625rem !default;
 $d-p-side-image-banner-half-padding: 5% !default; // Spacing between content and half of the screen
 
+.theme-default {
+  .paragraph--type--d-p-side-image {
+    color: $d-p-side-image-banner-text-color;
+
+    .field--name-field-d-main-title {
+      color: $d_p_side-image-header-color;
+    }
+
+    .image-side {
+      &-right,
+      &-right-wide {
+        background: $d-p-side-image-banner-background-color;
+      }
+
+      &-left,
+      &-left-wide {
+        background: $d-p-side-image-banner-background-color-secondary;
+      }
+    }
+
+    a:not(.btn) {
+      color: $d-p-side-image-banner-link-color;
+      &:hover {
+        color: $d-p-side-image-banner-link-hover-color;
+      }
+    }
+  }
+}
+
 // Override default values.
 .padding-top-default {
-  .paragraph.paragraph--type--d-p-side-image {
-    padding-top: 0;
+  .paragraph {
+    &.paragraph--type--d-p-side-image {
+      padding-top: 0;
+
+      .container-half {
+        padding-top: $d-p-side-image-banner-text-padding;
+      }
```

Comment:
hm, no to, to `padding-top: 0;` jest w sumie niepotrzebne dzieki temu, ze mamy nadaną klasę `content-moved-inside`.

a zamiast `container-half` moze uzywajmy tu klas analogicznie jak `padding-x-x`, czyli `content-inside-wrapper`. Poniżej też :pray: